apps/web/src/components/BridgeModal/BridgeForm.schema.ts

```diff
@@ -0,0 +1,20 @@
+import * as yup from 'yup'
+
+export interface BridgeFormValues {
+  amount?: number
+}
+
+const bridgeFormSchema = (userL1Balance: number) =>
+  yup.object({
+    amount: yup
+      .number()
+      .required()
+      .max(userL1Balance, 'Your balance is insufficient to send ETH.')
+      .test(
+        'is-greater-than-0',
+        'Must send more than 0 ETH',
+        (value) => !!value && value > 0
```

Comment:
why are we using max to detect if a balance is insufficient? and can we not use min to detect greater than 0? i think you should be able to supply a custom message if i'm not mistaken.

Reply:
because their balance on L1 is the max amount they can bridge. if they try to bridge 2 eth but they only have 1 eth for ex their balance is not sufficient to cover a 2 eth bridge call. these validation checks are pulled from the SendEth proposal template validation schema so the messages are a bit confusing can change those + the min check.

Reply:
tried making the switch to min and it seems to not validate for zero values ie if a user inputs `0` the min check passes. think this is the reason why the SendEth template used this custom test for min zero checks @nadiakiamilev

Reply:
ahhh sweet ok. thanks for reminding me it's been awhile 😅 all good.